proto/viam/app/data/v1/data.proto

```diff
@@ -242,6 +242,7 @@ message BinaryMetadata {
 // DeleteTabularDataByFilterRequest deletes the data and metadata of tabular data when a filter is provided
 message DeleteTabularDataByFilterRequest {
   Filter filter = 1;
+  bool include_internal_data = 2;
```

Comment:
+1, can remove here. We're removing DeleteTabularDataByFilter in favor of DeleteTabularData.

Reply:
kk removed the change to DeleteTabularDataByFilter!

Reply:
Actually, doesn’t this mean we wouldn’t be able to delete tabular data for a specific session? Should we just let any calls to DeleteTabularData also delete the internal data at that point?

Reply:
Good point, we won't be able to delete tabular data for a specific session. Yes, I think it's okay if we don't automatically delete tabular data after a session, and just let calls to DeleteTabularData clean it up later. But I'll check with Esha in the product meeting.

Reply:
Discussed in the SLAM product meeting that we won't attempt to delete tabular data after a session, since it is pretty negligible. However, DeleteTabularData should include internal data by default so that users can clean this up. Maybe in the future, we could introduce functionality to regularly clean up old internal data.

Reply:
updated to remove include_internal_data from `DeleteTabularDataRequest`, so now only DeleteBinaryDataByFilter will include the field

Reply:
we save the tabular data during a session, but we only use the most recently generated tabular data

Reply:
if it becomes an issue I think we can add in the functionality later

Reply:
LGTM if that edge case is ok for the SLAM team now.